client/src/components/Collections/PairedListCollectionCreator.vue

```diff
@@ -93,6 +93,14 @@ const hasFilter = computed(() => forwardFilter.value || reverseFilter.value);
 const strategy = ref(autoPairLCS);
 const duplicatePairNames = ref<string[]>([]);
 
+const canClearFilters = computed(() => {
+    return forwardFilter.value || reverseFilter.value;
+});
+
+const canAutoPair = computed(() => {
+    return forwardFilter.value && reverseFilter.value;
```

Comment:
```suggestion
    return forwardFilter.value && reverseFilter.value && pairableElements.value.length > 0;
```

Since an auto pair wouldn't be possible ofc with no pairable elements for the filters?